dao-user-reference.adoc

```diff
@@ -0,0 +1,192 @@
+
+==== Blind Voting Phase
+
+_450 blocks, about 3 days_
+
+You may approve, reject, or abstain from casting a vote on proposals. It is highly recommended that you only approve or reject proposals for which you can make a well-informed decision.
+
+It is not necessary to have made a proposal to be able to vote—any stakeholder (anyone with BSQ) can participate in voting.
+
+Once you have made your selections, you must specify a stake and publish your votes.
+
+Votes cannot be changed once they are published.
+
+===== Voting Stake
+
+Your voting weight is determined by 2 factors: _reputation_ (how much BSQ you've earned), and _stake_ (how much of your owned BSQ you allot to your vote) that you specify in your vote transaction.
+
+_Reputation_ is the amount of BSQ earned by doing work for the Bisq network, independent of the amount of BSQ spent or sold. So if Alice earns 10,000 BSQ by working on Bisq, and she sells 8,000 BSQ for BTC to pay her bills, her reputation value remains 10,000 BSQ. Caveat: reputation decays over time. BSQ earned decays linearly over a period of 2 years as soon as it's earned. So the reputation value of Alice's 10,000 BSQ is 5,000 BSQ one year after she earns it, and zero two years later.
+
+Reputation cannot be specified when making a vote transaction—it's automatically calculated for the user from their past contributions.
+
+_Stake_ is the literal amount of BSQ specified for the vote transaction. This is BSQ you own that you would like to devote to your vote. The more BSQ you allot, the more weight your vote will hold. BSQ specified for stake is unspendable for the duration of the voting cycle.
+
+**Total voting weight is the sum of reputation value and specified stake.**
+
+Reputation is worth more than stake. The idea is that there is no better measure of good-will and dedication to the project than actually working on the project. The arrangement still allows people who buy into the project considerable influence, while hopefully minimizing the potential of whales with bad intentions from buying their way in and sabotaging the project.
+
+===== Other Details
+
+Votes are published on the Bitcoin network and _encrypted_ vote data objects are broadcast on the Bisq P2P network (hence the term 'blind' vote). The Bitcoin transaction includes a hash of the vote data object. BSQ specified for voting stake is also specified in this transaction, and cannot be spent until the vote reveal transaction is published.
+
+Following the end of the blind voting phase, there is a break of 10 blocks to protect against reorgs.
+
+==== Vote Reveal Phase
+
+_450 blocks, about 3 days_
+
+No action is required of the user during this phase aside from having the Bisq software online at some point during the phase so the vote reveal phase so it can publish the vote reveal transaction.
+
+You should see a pop-up in the UI when the transaction has been published.
```

Comment:
-> You will see

Reply:
reports from users who have not seen it was because they did not publish the vote